### DISCUSSION TITLE
Harman car radio patch issues

### DISCUSSION THREAD

**Discussion by luigi1980** [Mr-MIBonk/M.I.B._More-Incredible-Bash#47] [General]
Hi I apologize in advance if I do something wrong please correct me ....... I state I have an Audi A4 2017 with car radio Harman MMI Plus, I installed the firmware MHI2Q_ER_AUG22_P5092_MU1329 and I tried to make the patch but to no avail, but not even the USB port works I installed a USB socket connected with the cable and power supply on the green plug 5V but nothing goes ...... Possible help?

**Comment by harman-f** [comment]
Which USB port is installed in your car?
Charge only does not support USB data.

**Comment by luigi1980** [comment]
Thank you..... However the USB data socket I installed I installed the single and does not work

**Reply by harman-f** [reply]
Please share your M.I.B log file

Looks like you did something wrong during installation of the USB socket

**Reply by luigi1980** [reply]
The USB socket is connected with the cable to the yellow plug of the MMI and the power supply connected to the green plug of the power supply which is 5V socket and the 5G0035222E ..... Thank you

**Comment by harman-f** [comment]
Something must be wrong on the hardware side.

Coding on MIB unit is always done correctly by M.I.B.

**Reply by luigi1980** [reply]
thanks for the answer, but the installation could not even before the installation of the USB .... anyway you can show me how to redo everything maybe try again and you turn the log here ? and then I have to wait when I launch the patch ? the screen becomes all black....

**Reply by harman-f** [reply]
SD card /backup/logs

M.I.B is running from SD card. Even without USB you can use and run M.I.B.
